src/fee/utils.ts

```diff
@@ -0,0 +1,124 @@
+import BigNumber from '@waves/bignumber';
+import { Asset, Money } from '@waves/data-entities';
+import { TRANSACTION_TYPE } from '@waves/ts-types';
+import { getMoney } from 'ui/utils/converters';
+import * as invokeScriptParseTx from '../ui/components/transactions/ScriptInvocation/parseTx';
+import * as transferParseTx from '../ui/components/transactions/Transfer/parseTx';
+import { DEFAULT_FEE_CONFIG } from '../constants';
+import { Message } from 'ui/components/transactions/BaseTransaction';
+import { AssetDetail } from 'ui/services/Background';
+import { AccountBalance, AssetBalance } from 'ui/reducers/updateState';
+import { SPONSORED_FEE_TX_TYPES } from './constants';
+
+export function convertFeeToAsset(fee: Money, asset: Asset) {
+  const minSponsoredFee = (asset: Asset) =>
+    asset.id === 'WAVES'
+      ? DEFAULT_FEE_CONFIG.calculate_fee_rules.default.fee
```

Comment:
I think we need to use the config from https://raw.githubusercontent.com/Waves-Keeper/configs/master/fee.json. If we want to change the value of the field, then there will be a conflict between these two configs.

Reply:
Moved fee config to remote config controller